README.md

```diff
@@ -752,12 +752,17 @@ Eio.Io Net Connection_failure Refused Unix_error (Connection refused, "connect",
 ```
 
 If we ran this using another backend, the `Unix_error` part might change.
-To avoid this problem, you can use `Eio.Exn.Backend.show` to hide the backend-specific part of errors:
+To avoid this problem, you can use `Eio.Exn.Backend.show`
 
 ```ocaml
 # Eio.Exn.Backend.show := false;;
 - : unit = ()
+```
+
+to hide the backend-specific part of errors:
 
+<!-- $MDX skip -->
+```ocaml
 # Eio_main.run @@ fun env ->
```

Comment:
For some reason this test never completes. So, I disabled it.

Reply:
That seems worrying! Is it only an MDX thing, or does the example in `examples/net` also hang if the server side isn't running?